src/kirby/components/card/card.component.tns.scss

```
@@ -4,13 +4,14 @@ $color-card-bg: get-color('kirby-contrast-light');
 
 .background, 
 .container {
+    display: flex;
     flex-direction: column;
-    width: 100%;
+    text-align: center;
+    margin: 8;
```

Here you should use `size('s')`.
```suggestion
    margin: size('s');
```